### PR TITLE
OSDOCS-4078: Welcome page updates

### PR DESCRIPTION
**Pull request by opayne1**
[OSDOCS-4078](https://issues.redhat.com//browse/OSDOCS-4078): Welcome page updates

Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4078

Link to docs preview:
https://54587--docspreview.netlify.app/openshift-enterprise/latest/welcome/index.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mburke5678** [comment]
Not sure where it would go in the list, but this might be good features to add. First is MCO, second is Node, if that matters

RHCOS image layering
Red Hat Enterprise Linux CoreOS (RHCOS) image layering allows you to add new images on top of the base RHCOS image. This layering does not modify the base RHCOS image. Instead, it creates a custom layered image that includes all RHCOS functionality and adds additional functionality to specific nodes in the cluster. For more information, see [RHCOS image layering](https://docs.openshift.com/container-platform/4.12/post_installation_configuration/coreos-layering.html#coreos-layering).

crun container runtime (Technology Preview)
OpenShift Container Platform now supports the crun container runtime in Technology Preview. You can switch between the crun container runtime and the default container runtime as needed by using a ContainerRuntimeConfig custom resource (CR). For more information, see [About the container engine and container runtime](https://docs.openshift.com/container-platform/4.12/nodes/containers/nodes-containers-using.html#nodes-containers-runtimes).

**Review by ShaunaDiaz — COMMENTED** [review]
Looks like lines 181 and 184 should be part of the bullet above or made into their own bullets.

**Review by mjpytlak — COMMENTED** [review]
@opayne1 Thanks for managing these updates. In addition to my one comment, I happened to miss an update to the "Install a private cluster" bullet. Please add a link for IBM Cloud VPC (installing/installing_ibm_cloud_public/installing-ibm-cloud-private.adoc#installing-ibm-cloud-private)

File: welcome/index.adoc [hunk 1]
```diff
@@ -83,7 +83,7 @@ You can also deploy a cluster on AWS infrastructure that you provisioned yoursel
 - **xref:../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[Install a cluster on GCP]**: You can deploy clusters with xref:../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[default settings] or xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[custom GCP settings] on Google Cloud Platform (GCP). You can also perform a GCP installation where you provision your own infrastructure.
 
 ifndef::openshift-origin[]
-- **xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[Install a cluster on IBM Cloud]**: You can install {product-title} on IBM Cloud on installer-provisioned infrastructure.
+- **xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[Preparing to install a cluster on IBM Cloud VPC]**: You can install {product-title} on IBM Cloud VPC on installer-provisioned infrastructure.
```
[the remark this answers is not in the document]
Comment (opayne1) [reply welcome/index.adoc:86]: Sure! Thats what I had it to at first, but wasn't sure. I will fix. Thanks for the review! 😄

**Review by JoeAldinger — COMMENTED** [review]
/lgtm
links all look good and I see that pesky `*`. Nice work!

**Comment by bergerhoffer** [comment]
This should have been submitted against main, trying to CP it there now. (Though it might fail since others have had to resolve conflicts in the file recently)

**Pull request merged**